atomic.h

```diff
@@ -1,203 +0,0 @@
-/*
- * Copyright (C) 2006 kernel.org
- *
- * This file is part of opensips, a free SIP server.
```

Comment:
Interesting stuff, @sobomax!  This header was originally added in 2006, since there was no libc support for these operations.  It is refreshing to find out that they've been available since 2011 and we can get away with maintaining less code :)